packages/ng/forms/abstract-field-component.ts

```diff
@@ -0,0 +1,17 @@
+import { booleanAttribute, Component, inject } from '@angular/core';
+import { injectNgControl } from './inject-ng-control';
+import { NG_VALIDATORS, RequiredValidator, Validators } from '@angular/forms';
+
+@Component({
+	template: '',
+	standalone: true,
+})
+export class AbstractFieldComponent {
+	protected ngControl = injectNgControl();
+
+	#ngModelRequiredValidator: RequiredValidator | null = inject(NG_VALIDATORS, { optional: true })?.find((v): v is RequiredValidator => v instanceof RequiredValidator);
```

Comment:
Could be `undefined` as well:

```suggestion
	#ngModelRequiredValidator: RequiredValidator | null | undefined = inject(NG_VALIDATORS, { optional: true })?.find((v): v is RequiredValidator => v instanceof RequiredValidator);
```